front/app/modules/commercial/analytics/admin/hooks/useProposals/parse.ts

```diff
@@ -0,0 +1,50 @@
+import { XlsxData } from 'components/admin/ReportExportMenu';
+import { ProposalsChartData } from './typings';
+import { formatCountValue } from '../../utils/parse';
+
+export const parseProposalsChartData = (
+  all,
+  allPeriod,
+  successful,
+  successfulPeriod
```

Comment:
Maybe instead of passing these all as seperate, untyped parameters, you could just pass in `Response['data']`? Adds type safety and reduces the number of parameters